Assets/StatusBar.cs

```diff
@@ -0,0 +1,68 @@
+﻿using UnityEngine;
+using System.Collections;
+
+public class StatusBar : MonoBehaviour {
+
+    private int current;
```

Comment:
current and max are descriptive names but they're don't read well in code. if you can find a better name that would be good, but i can't think of any so if you leave it like this it's fine too.